src/crd.rs

```diff
@@ -223,3 +223,11 @@ where
         .await;
     Ok(())
 }
+
+pub trait HasInstance {
```

Comment:
These should get some comments. Maybe add a todo if the comments become longer..

Reply:
added small comment.